README.md

```diff
@@ -127,3 +128,8 @@ Additional configuration
 You can build a statically linked toolchain by uncommenting
 `CT_STATIC_TOOLCHAIN=y` in the file `crosstool-config-overrides`. More
 fine-tunable options may be available in that file and/or Makefile.
+
+Testing your enviroment
+=======================
+There are several opensource examples out there, but you can clone one of these repos to verify your fresh new installation
+https://github.com/esp8266/source-code-examples
```

Comment:
I do not endorse non-opensource software. Any software which doesn't come with opensource license is not opensource. I opened https://github.com/esp8266/source-code-examples/issues/8 for this, feel free to watch that issue and make sure it comes to a resolution.